opensrp-reveal/src/main/java/org/smartregister/reveal/sync/RevealClientProcessor.java

```diff
@@ -76,21 +79,31 @@ public void processClient(List<EventClient> eventClients) {
 
     }
 
-    public void processSprayEvent(Event event) {
+    private void processSprayEvent(Event event, ClientClassification clientClassification) {
         if (event.getDetails() != null && event.getDetails().get(TASK_IDENTIFIER) != null) {
```

Comment:
Could event ever be null?

Reply:
This shouldn't be a null for A Spray event, this is a safe check

Reply:
ok.